zustand/zustand/index.js

```diff
@@ -0,0 +1,159 @@
+import AssignmentIcon from '@material-ui/icons/Assignment';
+import PersonIcon from '@material-ui/icons/Person';
+import SchoolIcon from '@material-ui/icons/School';
+import WorkIcon from '@material-ui/icons/Work';
+import { create } from 'zustand';
+import { devtools } from 'zustand/middleware';
+
+export const useResumeStore = create(
+  devtools(set => ({
+    data: {
+      personal: {},
+      resumeMeta: {},
+      experience: [],
+      extras: [],
+      education: [],
+    },
+    addpersonaldata: personal =>
+      set(state => ({
+        data: {
+          ...state.data,
+          personal: personal,
+        },
+      })),
+    addresumemetadata: meta =>
+      set(state => ({
+        data: {
+          ...state.data,
+          resumeMeta: meta,
+        },
+      })),
+
+    addexperiencedata: experience =>
+      set(state => ({
+        data: {
+          ...state.data,
+          experience: experience,
+        },
+      })),
+    addextrasdata: extras =>
+      set(state => ({
+        data: {
+          ...state.data,
+          extras: extras,
+        },
+      })),
+
+    deletesingleextra: id =>
+      set(state => ({
+        data: {
+          ...state.data,
+          extras: state.data.extras.filter(e => e.id !== id),
+        },
+      })),
+
+    addsampleextra: extras =>
+      set(state => ({
+        data: {
+          ...state.data,
+          extras: [...state.data.extras, extras],
+        },
+      })),
+
+    addeducationdata: education =>
+      set(state => ({
+        data: {
+          ...state.data,
+          education: education,
+        },
+      })),
+
+    deletesingleexperience: id =>
+      set(state => ({
+        data: {
+          ...state.data,
+          experience: state.data.experience.filter(a => {
+            console.log(a.id);
+            return a.id !== id;
+          }),
+        },
+      })),
+    deletesingleeducation: id =>
+      set(state => ({
+        data: {
+          ...state.data,
+          education: state.data.education.filter(a => {
+            return a.id !== id;
+          }),
+        },
+      })),
+
+    addsampleexperience: experience =>
+      set(state => ({
+        data: {
+          ...state.data,
+          experience: [...state.data.experience, experience],
+        },
+      })),
+    addsampleeducation: education =>
+      set(state => ({
+        data: {
+          ...state.data,
+          education: [...state.data.education, education],
+        },
+      })),
+    updatetitel: titel =>
```

Comment:
why is it named updatetitle1 ?

and variable names need to camelcase
as in updateTitle.